edb/pgsql/delta_ext_ai.py

```diff
@@ -61,6 +61,9 @@
 #                            -- embedding-text-3- models).
 #  )
 #
+# TODO: We could probably drop target_attr since the names should be
```

Comment:
Probably not, see review comment on `get_fts_index_id`.  We want to minimize the computation done in the indexer since CPU time there is a scarce resource.